addons/tutorial_04_split_tunneling/manifest.json

```diff
@@ -0,0 +1,94 @@
+{
+  "api_version": "0.1",
+  "id": "tutorial_04_split_tunneling",
+  "name": "Tutorial: Split tunneling",
+  "conditions": {
+    "enabled_features": ["splitTunnel"],
+    "javascript": "splitTunnelingOff.js"
```

Comment:
Just to confirm, we are requiring split tunneling to be turned off for the tutorial to even be visible to the user?

If so, I think we should update the A/C because right now it states: 

> If a user had already disabled the “Protect all apps with VPN” toggle, the tutorial should flip it back on so that the tutorial can advance on firm footing telling users to flip it off.